src/canary_server.cpp

```diff
@@ -362,6 +362,8 @@ void CanaryServer::loadModules() {
 	g_game().loadBoostedCreature();
 	g_ioBosstiary().loadBoostedBoss();
 	g_ioprey().InitializeTaskHuntOptions();
+	// Spawn monsters and npcs
+	g_game().map.spawnCreatures();
```

Comment:
Won't this spawn only main map?

Reply:
Yes, but I only disabled the loading for the main map.